core/templates/dev/head/components/ck-editor-helpers/ck-editor-4-rte.directive.ts

```diff
@@ -86,8 +87,10 @@ angular.module('oppia').directive('ckEditor4Rte', [
         }).join(',');
         var buttonNames = [];
         names.forEach(function(name) {
-          buttonNames.push('Oppia' + name);
-          buttonNames.push('-');
+          if (!(contextIsExplorationPlayer)) {
```

Comment:
Use canWriteToFs here. Delete contextIsExplorationPlayer altogether, here and above.